extensions/jdbc/jdbc-h2/runtime/src/main/java/io/quarkus/jdbc/h2/runtime/graal/CompareMode.java

```diff
@@ -1,39 +0,0 @@
-package io.quarkus.jdbc.h2.runtime.graal;
-
-import org.h2.engine.SysProperties;
-
-import com.oracle.svm.core.annotate.Alias;
-import com.oracle.svm.core.annotate.Substitute;
-import com.oracle.svm.core.annotate.TargetClass;
-
-@TargetClass(org.h2.value.CompareMode.class)
-public final class CompareMode {
```

Comment:
Looks like it's now working without this one. But I'm not completely sure this is tested by our test suite?